app/scripts/controllers/pod.js

```diff
@@ -218,9 +218,13 @@ angular.module('openshiftConsole')
       $scope.containersEnv = containers;
     };
 
+    var annotation = $filter('annotation');
     var podResolved = function(pod, action) {
       $scope.loaded = true;
       $scope.pod = pod;
+      $scope.dcName = annotation(pod, 'deploymentConfig');
+      $scope.rcName = annotation(pod, 'deployment');
+      $scope.deploymentVersion = annotation(pod, 'deploymentVersion');
```

Comment:
> or are we waiting for the rebase to get pods with ownerRefs

Yes, waiting for owner refs. I don't see any way to do this for other controllers yet.